packages/functions/src/index.ts

```diff
@@ -3,8 +3,53 @@ import * as functions from "firebase-functions";
 import {checkAuthToken} from "./middlewares/check-auth-token";
 import authRouter from "./resources/auth/auth.router";
 import userRouter from "./resources/user/user.router";
+import * as admin from "firebase-admin";
```

Comment:
let's not use import  * when it's possible, for tree-shaking purpose:

`import { initializeApp, firestore } from "firebase-admin";`